client/src/components/Assets/Icons/index.tsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+
+export const encodeSVG = (svg: string) => encodeURI(svg).replace(/#/g, '%23');
+
+export type IconProps = {
+  alt?: string;
+  fill?: string;
+  height?: string;
+  style?: object;
+  width?: string;
+  className?: string;
+  svgClass?: string;
+};
+
+export type BaseSvgProps = {
+  svg: string;
+  alt?: string;
+  fill?: string;
+  height?: string;
+  size?: string;
+  style?: object;
+  width?: string;
+};
+
+export const BaseSvg = ({
+  svg,
+  size = '1em',
+  height = size,
+  width = size,
+  style,
+  alt = '',
+  ...props
+}: BaseSvgProps) => (
+  <img
+    src={`data:image/svg+xml;utf8,${encodeSVG(svg)}`}
+    style={{
+      ...style,
+      width,
+      height,
+    }}
+    alt={alt}
+    {...props}
```

Comment:
(Optional): {...props} is bad generally. For this case, it's ok tough, but I think it would be better if you could list all the needed props explicitly.